### PR TITLE
UI - Application Details Bug Fixes

### PR DESCRIPTION
**Pull request by shaangill025**
*Issue:* https://github.com/bcgov/entity/issues/22583
https://github.com/bcgov/entity/issues/22594

*Description of changes:*
Included fixes and match mobile Figma design for application details page.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namex license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://strr-ui-dev--pr-104-ccthlhy0.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://strr-ui-dev--pr-104-ccthlhy0.web.app

File: strr-web/pages/application-details/[id]/index.vue [hunk 1]
```diff
@@ -283,4 +286,37 @@ const getContactRows = (contactBlock: ContactI) => [{
   SIN: contactBlock.socialInsuranceNumber,
   'BN (GST)': contactBlock.businessNumber
 }]
+
+const bannerRef = ref(null)
+const contentRef = ref(null)
+const isMobile = ref(false)
+
+const updatePadding = async () => {
```
[the remark this answers is not in the document]
Comment (shaangill025) [reply strr-web/pages/application-details/[id]/index.vue:294]: Yes for 22583, it is to adjust the content div top padding according to the height of the banner. Now with mobile, the banner wraps and height changes dynamically, padding of the div below adjust to it [consistency, 0.5 x height of the banner].

[the remark this answers is not in the document]
Comment (dimak1) [reply strr-web/pages/application-details/[id]/index.vue:294]: Is there a tailwind/css only solution?

[the remark this answers is not in the document]
Comment (shaangill025) [reply strr-web/pages/application-details/[id]/index.vue]: I've removed the dynamic padding implementation and now has a tailwind/css solution that is reasonable.

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://strr-ui-dev--pr-104-ccthlhy0.web.app

**Review by dimak1 — APPROVED** [review]
Great, thanks for the fix via css only. Looks good to me.

**Pull request merged**